RevenueCatUI/Templates/MultiPackageTemplate.swift

```diff
@@ -0,0 +1,241 @@
+                }
+                    .scrollBounceBehaviorBasedOnSize()
+            }
+
+            self.subscribeButton
+                .padding(.bottom)
+                .padding(.horizontal)
+        }
+        .animation(.easeInOut(duration: 0.1), value: self.selectedPackage)
+        .frame(maxHeight: .infinity)
+        .multilineTextAlignment(.center)
+        .frame(maxHeight: .infinity)
+    }
+
+    private var scrollableContent: some View {
+        VStack {
+            Text(self.selectedLocalization.title)
+                .font(.largeTitle.bold())
+
+            Spacer()
+
+            Text(self.selectedLocalization.subtitle)
+                .font(.title2)
+
+            Spacer()
+
+            self.packages
+
+            Spacer()
+        }
+        .padding(.horizontal)
+        .frame(maxHeight: .infinity)
+    }
+
+    private var packages: some View {
+        VStack(spacing: 8) {
+            ForEach(self.configuration.packages.all, id: \.content.id) { package in
+                Button {
+                    self.selectedPackage = package.content
+                } label: {
+                    self.packageButton(package, selected: self.selectedPackage === package.content)
+                }
+                .buttonStyle(PackageButtonStyle())
+            }
+        }
+        .padding(.bottom)
+    }
+
+    @ViewBuilder
+    private func packageButton(_ package: TemplateViewConfiguration.Package, selected: Bool) -> some View {
+        let alignment: Alignment = .leading
+
+        VStack(alignment: alignment.horizontal, spacing: 5) {
+            HStack {
+                Image(systemName: "checkmark.circle.fill")
+                    .foregroundColor(
+                        selected
+                        ? self.configuration.colors.callToActionBackgroundColor
+                        : .gray
+                    )
+
+                Text(package.content.productName)
+            }
+            .foregroundColor(self.configuration.colors.callToActionBackgroundColor)
+
+            IntroEligibilityStateView(
+                textWithNoIntroOffer: package.localization.offerDetails,
+                textWithIntroOffer: package.localization.offerDetailsWithIntroOffer,
+                introEligibility: self.introEligibility[package.content],
+                foregroundColor: selected
+                    ? .white
+                    : .black,
+                alignment: alignment
+            )
+            .fixedSize(horizontal: false, vertical: true)
+            .font(.body)
+        }
+        .font(.body.weight(.medium))
+        .padding()
+        .multilineTextAlignment(.leading)
+        .frame(maxWidth: .infinity, alignment: alignment)
+        .foregroundColor(self.configuration.colors.foregroundColor)
+        .background {
+            RoundedRectangle(cornerRadius: 15, style: .continuous)
+                .foregroundColor(selected ? .black : .init(white: 0.8))
```

Comment:
Still need to improve these colors. Once this template is in the frontend we can talk about which new color "definitions" we need /cc @lburdock @guido732